src/sentry/api/endpoints/relocations/index.py

```diff
@@ -107,7 +107,7 @@ def post(self, request: Request) -> Response:
         validated = serializer.validated_data
         fileobj = validated.get("file")
         owner_username = validated.get("owner")
-        org_slugs = validated.get("orgs")
+        org_slugs = validated.get("orgs").split(",")
```

Comment:
Some test cases I think we should consider adding:
"foo,bar," (should be okay)
"foo,,bar"
",,"
"foo,     bar"
"foo,\nbar"
"foo\nbar"
"foo bar"

We could probably just make 1 more test method that loops over these and ensures that it always gets the same 400 response for each.